.github/workflows/ci.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       - name: Install playwright browsers
         run: npx playwright install --with-deps
       # run yarn and pass through `-- --all` to turborepo
-      - run: yarn test -- -- --all
+      - run: yarn test -- --concurrency=1 -- --all
```

Comment:
this unblocks the PR.
Locally, for me just `yarn test` was mostly succeeded. Also, running individual projects succeeded without issues. Only running `yarn test -- -- --all` failed.
I think it's a reasonable change to unblock this PR. We'd still want to look at addressing the root cause and fixing it to be able to remove this concurrency limitation.
Test timing goes from ~35s to ~70s.